server/startup/cdn.js

```diff
@@ -0,0 +1,13 @@
+import { Meteor } from "meteor/meteor";
+
+/**
+ * Sets prefix for the user who wants to serve bundled js and css
```

Comment:
This is not in the correct JSDoc format. Please look at the documentation for the correct format.

Also, javascript functions don't return `null` by default they return `undefined` which is not the same thing